docs/modules/ROOT/pages/api/erc20.adoc

```diff
@@ -0,0 +1,431 @@
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+:eip20: https://eips.ethereum.org/EIPS/eip-20[EIP-20]
+:erc20-guide: xref:erc20.adoc[ERC20 guide]
+:casing-discussion: https://github.com/OpenZeppelin/cairo-contracts/discussions/34[here]
+
+= ERC20
+
+Reference of interfaces and utilities related to ERC20 contracts.
+
+TIP: For an overview of ERC20, read our {erc20-guide}.
+
+== Core
+
+[.contract]
+[[IERC20]]
+=== `++IERC20++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/cairo-2/src/token/erc20/interface.cairo#L6-L19[{github-icon},role=heading-link]
+
+[.hljs-theme-dark]
+```javascript
+use openzeppelin::token::erc20::interface::IERC20;
+```
+
+Interface of the IERC20 standard as defined in {eip20}.
+
+[.contract-index]
+.Functions
+--
+* xref:#IERC20::name[`++name()++`]
+* xref:#IERC20::symbol[`++symbol()++`]
+* xref:#IERC20::decimals[`++decimals()++`]
+* xref:#IERC20::total_supply[`++total_supply()++`]
+* xref:#IERC20::balance_of[`++balance_of()++`]
+* xref:#IERC20::allowance[`++allowance(owner, spender)++`]
+* xref:#IERC20::transfer[`++transfer(recipient, amount)++`]
+* xref:#IERC20::transfer_from[`++transfer_from(sender, recipient, amount)++`]
+* xref:#IERC20::approve[`++approve(spender, amount)++`]
```

Comment:
```suggestion
* xref:#IERC20-name[`++name()++`]
* xref:#IERC20-symbol[`++symbol()++`]
* xref:#IERC20-decimals[`++decimals()++`]
* xref:#IERC20-total_supply[`++total_supply()++`]
* xref:#IERC20-balance_of[`++balance_of()++`]
* xref:#IERC20-allowance[`++allowance(owner, spender)++`]
* xref:#IERC20-transfer[`++transfer(recipient, amount)++`]
* xref:#IERC20-transfer_from[`++transfer_from(sender, recipient, amount)++`]
* xref:#IERC20-approve[`++approve(spender, amount)++`]
```

This is the pattern I've been using (the one from Solidity doc site), and I think is more URL friendly than the colons. This ids are shown in the URL.